### PR TITLE
TRITON-2019 agent-installer image name repeats itself repeats itself

### PR DESCRIPTION
**Pull request by timfoster**
Testing notes are in the ticket

### CODE REVIEW

File: mk-agents-shar [hunk 1]
```diff
@@ -99,6 +99,7 @@ function usage() {
     echo "          Typically the name of the repo branch. Note this can"
     echo "          be multiple space-separated build names (e.g. "
     echo "          'feature1 master'). Each name is attempted to find an "
+    echo "  -B      branch name to use for the generated shar"
```
Comment (trentm) [inline mk-agents-shar]: Did you mean to insert this one line lower, below the end of the `-b` help text?

File: mk-agents-shar [hunk 2]
```diff
@@ -99,6 +99,7 @@ function usage() {
     echo "          Typically the name of the repo branch. Note this can"
```
Comment (trentm) [inline mk-agents-shar]: `SDCINSTALLER_BRANCH= ...` on line 77 is now unused I think, and could be dropped.

File: mk-agents-shar (hunk 1 above)
Comment (timfoster) [reply mk-agents-shar]: Och, yes - thanks, I'll fix that.

File: mk-agents-shar (hunk 2 above)
Comment (timfoster) [reply mk-agents-shar]: Yep, thanks.

**Pull request merged**